doc/support-matrix.md

```diff
@@ -0,0 +1,50 @@
+# Software Support Matrix
+
+## Release Process
+
+RealSense devices pass validation and stability cycles. Every firmware + librealsense release is going through several levels of validation, from automated unit-tests to longer validation cycles.  
+
+After passing unit-tests and basic functional testing release is made available under [librealsense/releases](https://github.com/IntelRealSense/librealsense/releases) and marked as a ![pre-release](./img/prerelease.png).
+
+Once full validation report is internally reviewed and approved, release is promoted to ![latest](./img/latest_release.png) and published at [www.intelrealsense.com/developers/](https://www.intelrealsense.com/developers/).
+
+We recommend certain combinations (see **RECOMMENDED CONFIGURATIONS** at the link above) of firmware and software we have tested, but there is significant effort to keep maximum backward and forward compatibility between firmware and software versions.
+
+Validation is done on i86 NUC machines running Windows 10 and Ubuntu 16.04 LTS.
+
+Additional platforms are tested and supported on demand, driven by community requests and feedback.
+
+## D400 Stereoscopic Depth Cameras
+
+|                                | Win 7                  | Win 10 & 8.1                      | Ubuntu 16/18 LTS x64                       | Ubuntu Arm (Raspberry Pi, RK3399…) | NVidia Jetson Series | Android (7+) & JAVA | Mac OS            | Viewer         | C/C++       | Python      | C#  | ROS                | Node.js             | LabView        | Unity  | Matlab  | OpenNi2| Unreal|
```

Comment:
I'm not sure about HID sensor with  win 8.1. Prefer to mark it as not tested


Typo  - DMFT 
>Yes (requires script or DFMT INF)